src/extensibility/ExtensionManagerView.js

```diff
@@ -250,6 +250,12 @@ define(function (require, exports, module) {
                             info.metadata[prop] = info.metadata["package-i18n"][locale][prop];
                         }
                     });
+                    if (info.metadata["package-i18n"][locale].hasOwnProperty("author")) {
+                        info.metadata.author = info.metadata["package-i18n"][locale].author;
+                    }
+                    if (info.metadata["package-i18n"][locale].hasOwnProperty("contributors")) {
+                        info.metadata.contributors = info.metadata["package-i18n"][locale].contributors;
+                    }
```

Comment:
Btw, we may want to add a comment here explaining that `"author"` is safe for localization because it's just a "friendly name" and the GitHub user id is stored in a separate `"owner"` field (which will never be localized).